docs.sh

```diff
@@ -1,13 +1,38 @@
 #!/bin/bash
+
+# Insall gh-pages-multi
+npm i -g @koumoul/gh-pages-multi
+
+export PROJECT="camunda-ext"
+
+export VERSION=
```

Comment:
why do you need empty value?

Reply:
There are 3 main cases which this script can handle.
1. New release is issued by pushing a tag. TravisCI executes pipeline to build new release, there is non-empty TRAVIS_TAG env variable like 1.4.1 which then will be passed as folder name into gh-pages branch. So you'll have gh-pages branch with folders 1.4, 1.4.1, and you can open https://latera.github.io/camunda-ext/1.4 and https://latera.github.io/camunda-ext/1.4.1 to see specific version docs. So in this case VERSION variable will be not empty.
2. New commit is pushed to master branch. TravisCI executes pipeline to build new release, there is a TRAVIS_BRANCH env variable with "master" value. It will lead for updating "latest" folder content in gh-pages branch, so you can open https://latera.github.io/camunda-ext/latest to see master docs.  So in this case VERSION variable will be not empty.
3. Other, like pull request or generating docs on local machine. Documentation will be built for current commit and stored in docs local folder. gh-pages branch will not be updated. In this case VERSION variable will be empty.